win/main.go

```diff
@@ -49,20 +53,38 @@ func main() {
 	ctx, cancel := context.WithCancel(context.Background())
 	cmd := runServer(ctx)
 
-	ui, err := lorca.New("http://localhost:8088", "", 1024, 768)
+	defer func() {
+		log.Println("Cleanup")
+		cancel()
+		if err := cmd.Process.Kill(); err != nil {
+			log.Fatal("failed to kill server process: ", err)
+		}
+
+		if r := recover(); r != nil {
+			log.Fatal("ERROR: ", r)
+		}
+	}()
+
+	ui, err := lorca.New("", "", 1024, 768)
 	if err != nil {
+		log.Panic(err)
```

Comment:
Can we "fail more gracefully here?, something like printing a regular error?

Reply:
The idea here was to use Panic like a throw, so we can catch and run cleanup (Fatal was bypassing cleanup). Cleanup then recovers, then prints the error. Is there a better pattern to use?